mlflow/pytorch/__init__.py

```diff
@@ -536,6 +755,44 @@ def load_model(model_uri, **kwargs):
     :param kwargs: kwargs to pass to ``torch.load`` method.
     :return: A PyTorch model.
 
+    .. code-block:: python
+        :caption: Example
+
+        import torch
+        import mlflow
+        import mlflow.pytorch
+        # Set values
+        model_path_dir = ...
+        run_id = "96771d893a5e46159d9f3b49bf9013e2"
+        pytorch_model = mlflow.pytorch.load_model("runs:/" + run_id + "/" + model_path_dir)
+        y_pred = pytorch_model(x_new_data)
+    """
+
+    torch_model_artifacts_path = _get_model_artifact_path(model_uri)
+    return _load_model(path=torch_model_artifacts_path, **kwargs)
+
+
+def _get_model_artifact_path(model_uri):
```

Comment:
I have refactored the load_model into two parts, so that we dont duplicate the code in `load_state_dict`

Reply:
Do we still need `_get_model_artifact_path`?

Reply:
@shrinath-suresh Can we revert the changes on `load_model`?